File: Backend/Entity/Producto.cs
namespace Entity
{
    public class Producto
    {
        public string Id { get; set; }
        public string Nombre { get; set; }
        public string IdCompania { get; set; }
        public string IdProceso { get; set; }
        public bool Estado { get; set; }
        public bool Eliminado { get; set; }
        public DateTime Fecha_log { get; set; }

        public Producto() { }
    }
}

File: Frontend/src/types/Producto.ts
export type producto = {
    id: string;
    nombre: string;
    idProyecto: string;
    idProceso: string;
    estado: boolean;    
    fecha_log: string;
  }

  export type Regisproducto = {
    id: string;
    idProduct: string;
    idRegisOrden: string;
    idProyecto: string;
    idUsuario: string;
    estado: boolean;    
    fecha_log: string;
  }

  export type Camp = {
    id: string;
    nombre: string;
    tipoDato: string;
    obligatorio: boolean;    
    idProduct: string;
    estado: boolean;
    fecha_log: string; // Usar string si la fecha se maneja como cadena de texto
  }

  export type oCampV = {
    id: string;
    valor: string;
    idProductCamp: string;
    idRegisProduct: string;  
    estado: boolean;  
    fecha_log: string; // Usar string si la fecha se maneja como cadena de texto
  }

  export type create_Regis = {
    producto: Regisproducto;
    campos: oCampV[];
  }

  export type create_Product = {
    producto: producto;
    campos: Camp[];
  } 